frontend/src/entities/dataset/dataset.service.tsx

```diff
@@ -144,6 +146,17 @@ export async function uploadResources (datasetContext: DatasetContext, resourceO
 
             if (uploadResponse.status === 204) {
                 successCount++;
+                notificationService.generateNotification(
+                    `Successfully uploaded ${successCount}/${resourceObjects.length} file(s).`,
+                    'in-progress',
+                    'upload-notification',
+                    (<ProgressBar
```

Comment:
Very cool! Love this! We may be able to use this in more places around the application.